### PR TITLE
Bump modules to k8s 0.26

### PR DESCRIPTION
**Pull request by manno**
refers to https://github.com/rancher/fleet/issues/2084

### CODE REVIEW

File: charts/fleet-crd/templates/crds.yaml [hunk 1]
```diff
@@ -2435,6 +2435,15 @@ spec:
               agentResources:
                 nullable: true
                 properties:
+                  claims:
```
Comment (weyfonk) [inline charts/fleet-crd/templates/crds.yaml:2438]: :thinking: where does this come from?

**Review by weyfonk — COMMENTED** [review]
LGTM with one doubt

**Review by weyfonk — APPROVED** [review]

**Comment by kkaempf** [comment]
Do we need a forward port to 0.9 / 0.10 ? 🤔

**Comment by weyfonk** [comment]
> Do we need a forward port to 0.9 / 0.10 ? 🤔

It doesn't seem so, see [this commit](https://github.com/rancher/fleet/commit/f8f5b004dd57a82c47db10ee3e2125392be89e61#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6) on `master` and [this one](https://github.com/rancher/fleet/commit/261d95861369663e58b585c503560bef9e7e2683#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6) on `release/v0.9`.

**Comment by olblak** [comment]
Shouldn't this pull request be named "Bump modules to k8s v0.27.9"

**Pull request merged**